src/bidiMapper/domains/input/ActionDispatcher.ts

```diff
@@ -550,8 +554,8 @@ export class ActionDispatcher {
           'Input.dispatchMouseEvent',
           {
             type: 'mouseWheel',
-            deltaX,
-            deltaY,
+            deltaX: deltaX * devicePixelRatio,
```

Comment:
we need to device by devicePixelRatio here more likely to account for the `* devicePixelRatio` on the CDP side